docs/integrations-monitoring.mdx

```diff
@@ -0,0 +1,71 @@
+# Integrations monitoring
+
+Superface comes with out-of-the-box monitoring of integrations used by your App. With Superface project you will get insights on your App's API dependencies and their health status. In case of use-case failure, you will also receive an email notification with details about the failing API calls.
+
+## Set up integrations monitoring
+
+This guide assumes you have created a Node.js app with first integration. If you missed that, see the previous step: [Getting&nbsp;started](./getting-started.mdx)
+
+<div class="count-steps-h3">
+
+### Create new project
+
+Create a new project in [Superface dashboard](/dashboard).
```

Comment:
Is this link correct? Even on production it will result in `superface.ai/docs/dashboard`, no?

Reply:
It is, `/docs/dashboard` points to a redirect: https://github.com/superfaceai/docs/blob/01479bb22aba1587b9d2cfd9c886ae70121fbca3/src/pages/dashboard.js

It's a hack devised by @kysely to avoid marking external links as external.